### PR TITLE
If cmake3 is found an >= 3.5 all is fine. When not try to use normal …

### PR DESCRIPTION
**Pull request by tuxmaster5000**
Fix for https://github.com/pytorch/pytorch/issues/36572#issuecomment-613500270

### CODE REVIEW

File: tools/setup_helpers/cmake.py [hunk 1]
```diff
@@ -116,12 +116,9 @@ def _get_cmake_command():
             return cmake_command
         cmake3 = which('cmake3')
         if cmake3 is not None:
-            cmake = which('cmake')
-            if cmake is not None:
-                bare_version = CMake._get_version(cmake)
-                if (bare_version < LooseVersion("3.5.0") and
-                        CMake._get_version(cmake3) > bare_version):
-                    cmake_command = 'cmake3'
```
Comment (xuhdev) [inline tools/setup_helpers/cmake.py:124]: Good catch. I think the original idea was that `cmake` should be used if it is >= 3.5.0, even if `cmake3` is found. Could you keep the change consistent with what was originally purposed?

Comment (tuxmaster5000) [reply tools/setup_helpers/cmake.py:124]: Do you mean something like the last PR?

Comment (xuhdev) [reply tools/setup_helpers/cmake.py:124]: What do you mean?

Comment (tuxmaster5000) [reply tools/setup_helpers/cmake.py:124]: The last changes at the PR from yesterday.

Comment (xuhdev) [reply tools/setup_helpers/cmake.py:124]: My comment is on the current change (the diff in this PR). There are two issues (if I'm not mistaken):

- If both `cmake` and `cmake3` are present, but `cmake3 < 3.5.0` and `cmake >= 3.5.0` the logic of the current change will lead to error.
- If both `cmake` and `cmake3` are present, and both `>=3.5.0`, `cmake3` will be returned, not `cmake`.

**Pull request closed without merging**